irods/pool.py

```diff
@@ -77,6 +81,12 @@ def get_connection(self):
 
             self.active.add(conn)
 
+            sess = self.session_ref()
+            if sess and sess.ticket__:
+                if not sess.ticket_applied.get(conn,False):
```

Comment:
Any reason these `if`s are split up?

Reply:
Don't know, I may have been putting a breakpoint there.  I certainly don't mind re-unifying it.